### ISSUE TITLE
getAccessToken() has a different API on iOS and Android

### ISSUE THREAD

**Issue by bllanos** [react-native-mapbox-gl/maps#937]
**Describe the bug**
On Android, `MapboxGL.getAccessToken()` returns an object with an `accessToken` property. On iOS, the same method returns a string.

**To Reproduce**

Example:
```js
import MapboxGL from '@react-native-mapbox-gl/maps';

MapboxGL.setAccessToken(config.get('accessToken'));
(async () => {
  try{
      let token = await MapboxGL.getAccessToken();
      console.log("Token: ", token);
  } catch (err) {
      console.error("Error checking Mapbox token:", err);
  }
})();
```
- On Android, the output is `Token:  {"accessToken": "<TOKEN>"}`
- On iOS, the output is `Token: <TOKEN>`

**Expected behavior**
`token` should be identical across Android and iOS.

**Versions**
 - Platform: Android, iOS
 - Device: Google Pixel, iPhone 8
 - Emulator/ Simulator: Emulator and physical Android and iOS devices
 - OS: iOS 13.5.1, Android 10 build number QP1A.191005.007.A3
 - react-native-mapbox-gl Version 8.1.0-rc.1
 - React Native Version 0.62.1

**Comment by mfazekas** [comment]
I'd say the iOS version is the correct one. 

Technically the fix is a breaking change. Or we can return a `String` instance with `accessToken` property also set

https://github.com/react-native-mapbox-gl/maps/blob/8f9170e78939dc22fa8135cc1aba26d9af0bbdc4/android/rctmgl/src/main/java/com/mapbox/rctmgl/modules/RCTMGLModule.java#L326-L331

**Comment by bllanos** [comment]
In the sense of the code interface, changing what the `getAccessToken` function returns is a breaking change, but as I haven't found the interface of the function in documentation pages in this repository, technically the change would not break any documented interface.

With respect to returning a `String`, there could be some unexpected side effects.
```js
let token = new String("test_token");
token.accessToken = "test_token_property";
token === "test_token"; // false
token == "test_token"; // true
let emptyToken = new String();
if(emptyToken) {
  console.log("Token is not empty"); // Executed
}
```

**Comment by mfazekas** [comment]
@bllanos sure i guess it's fine to modify android to call resolve with a `string` instead of object. IMO it's a potential breaking change but it's a rather insignificant one.

**Comment by bllanos** [comment]
Fixed by #954
